bot/seasons/evergreen/bookmark.py

```diff
@@ -0,0 +1,76 @@
+import logging
+
+import discord
+from discord.ext import commands
+
+log = logging.getLogger(__name__)
+
+
+class Bookmark(commands.Cog):
+    """A cog for Bookmarking a message."""

```

Comment:
Cog docstrings should be descriptive. This tells me nothing that I didn't already learn from `class Bookmark(commands.Cog)`. Explain what this cog is for, and how it works. Explain how to interact with it. This docstring serves as documentation for both users and programmers.